### ISSUE TITLE
Home Assistant Update Script Failing With Error

### ISSUE THREAD

**Issue by gcortes** [community-scripts/ProxmoxVE#628]
### Please verify that you have read and understood the guidelines.

Yes

### A clear and concise description of the issue.

When I run the update command in my Home Assistant LXC, I get the following output:

Updating to Stable Version
 ✓ Stopped Home Assistant
 / Updating Home Assistant   
[ERROR] in line 90: exit code 0: while executing command uv pip install ${BR}--upgrade homeassistant &> /dev/null

root@homeassistant-core:~# 

I've run the update command several time since I created the LXC without issue. It's only command I've run in the LXC.
I haven't done any updates to the LXC OS since I created it.

The update script contains one line:
bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/homeassistant-core.sh)"

If someone can tell me how to run verbose mode, I'll do so.

### What settings are you currently utilizing?

- [X] Default Settings
- [ ] Advanced Settings

### Which Linux distribution are you employing?

Debian 12

### Please provide detailed steps to reproduce the issue.

Enter the command update
Answer Update Core Only
Chose no to use Beta Branch

**Comment by tjcomserv** [comment]
Hi @gcortes , please can you go to the container shell, try the following and let us know the output?

```
systemctl stop homeassistant
source /srv/homeassistant/bin/activate
uv pip install --upgrade homeassistant
```

once done, type the following to get homeassistant up and running again:
```
systemctl start homeassistant
exit
```

**Comment by gcortes** [comment]
> Hi @gcortes , please can you go to the container shell, try the following and let us know the output?
> 
> ```
> systemctl stop homeassistant
> source /srv/homeassistant/bin/activate
> uv pip install --upgrade homeassistant
> ```
> 

This is what I got:

Updating to Stable Version
 ✓ Stopped Home Assistant
 / Updating Home Assistant   
[ERROR] in line 90: exit code 0: while executing command uv pip install ${BR}--upgrade homeassistant &> /dev/null

root@homeassistant-core:~# systemctl stop homeassistant
root@homeassistant-core:~# source /srv/homeassistant/bin/activate
(homeassistant) root@homeassistant-core:~# uv pip install --upgrade homeassistant
-bash: uv: command not found
(homeassistant) root@homeassistant-core:~# 


I went to update my notes on the start and stop command and found this:

cd /srv/homeassistant
source bin/activate
pip3 install --upgrade homeassistant

It's been sometime since I did an update so I don't remember if I used this before. Would it work after the scripts have been moved?

**Comment by tjcomserv** [comment]
Might be worth trying to install uv. 

`curl -LsSf https://astral.sh/uv/install.sh | sh`

This is from:
[https://docs.astral.sh/uv/#highlights](url)

**Comment by gcortes** [comment]
I decided to go with the pip3 install. I didn't use uv it because I wasn't sure how it would work with an environment set up with another tool.

It looks to have worked as Home Assistant came back up after the upgrade. I did get an error:
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
aiohttp-fast-url-dispatcher 0.3.0 requires aiohttp<3.10,>=3.8.5, but you have aiohttp 3.10.11 which is incompatible.
```

I also told me that I had an older version of pip:
```
[notice] A new release of pip is available: 24.0 -> 24.3.1
[notice] To update, run: pip install --upgrade pip
```
I update it as well. I'll see if that resolves the error when I upgrade again.

You can close this issue as far as I'm concerned. You might want to make note on the web page for Home Assistant that "update" doesn't work on older installs that don't have uv.
